README.md

```diff
@@ -150,25 +157,26 @@ address that your proxy is pointing to, you can use the `target` command in
 the CLI:
 
 ```sh
-web3 contract target --address 0xCONTRACT_ADDRESS
-0xTARGET_ADDRESS
+web3 contract target
```

Comment:
Still need TARGET?

Reply:
Ya, this returns the targeted contract address.

Reply:
Oh I thought it was an arg on the input.